### PR TITLE
BLASER readme and IWSLT example

### PR DESCRIPTION
**Pull request by Mortimerp9**
Create a base setup to run BLASER for IWSLT. This should make it clearer how to setup everything to run it.

### CODE REVIEW

File: demo/iwslt_blaser_eval/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+A lot of config is preset in `conf/eval_blaser.yaml`, please see the comments in that configuration
+file if you need to adjust anything. After that you can run the following command:
+
+```bash
+DEMO_DIR=<ENTERPATHTOIWSLTBLASEREVALDIRECTORY> python -m stopes.pipelines.eval.eval_blaser --config-dir $DEMO_DIR/conf tgt_manifest=<PATHTOTARGETMANIFEST.tsv> ref_manifest=<PATHTOREFERENCEMANIFEST.tsv>
+```
+
+Make sure to replace:
+
+- `ENTERPATHTOIWSLTBLASEREVALDIRECTORY` to the directory where you've run prepare.sh and downloaded everything
```
Comment (alexmourachko) [inline demo/iwslt_blaser_eval/README.md:67]: are we forced to use these filenames without underscores?

**Review by alexmourachko — APPROVED** [review]

File: demo/iwslt_blaser_eval/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+conda activate blaser
+
+# upgrade pip
+python -m pip install --upgrade pip
+
+# install fairseq
+pip install fairseq==0.12.1
+
+# install stopes+blaser 
+pip install 'stopes[blaser]'
```
[the remark this answers is not in the document]
Comment (Mortimerp9) [reply demo/iwslt_blaser_eval/README.md:22]: straight with PyPi (I've tested these commands on a fresh repo)

File: demo/iwslt_blaser_eval/README.md (hunk 1 above)
Comment (Mortimerp9) [reply demo/iwslt_blaser_eval/README.md:67]: no, it's just a placeholder :)

File: demo/iwslt_blaser_eval/mk_manifest.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+def audio_len(file: Path) -> tp.Tuple[Path, int]:
+    with contextlib.closing(wave.open(str(file), "r")) as wav_file:
+        frames = wav_file.getnframes()
+        rate = wav_file.getframerate()
+        duration = int(math.ceil(frames * 1000 / rate))
+        return (file, duration)
+
+
+def manifest(input_dir: Path, num_cpu: int = 4):
```
[the remark this answers is not in the document]
Comment (Mortimerp9) [reply demo/iwslt_blaser_eval/mk_manifest.py:26]: I think that multiprocessing will start 4 workers, even if you have 1 cpu. It won't crash, but there won't be much parallelism.

**Pull request merged**